dScripts/02_server/Enemy/AG/BossSpiderQueenEnemyServer.cpp

```diff
@@ -114,7 +106,10 @@ void BossSpiderQueenEnemyServer::WithdrawSpider(Entity* self, const bool withdra
 
 		//TODO: Set faction to -1 and set immunity
 		destroyable->SetFaction(-1);
-		destroyable->SetIsImmune(true);
+		GameMessages::SendSetStatusImmunity(
+			self->GetObjectID(), eStateChangeType::PUSH, self->GetSystemAddress(),
+			true, true, false, false, true
```

Comment:
What is the motivation behind using some of these defaults?  Assuming it is because of the client script?

Reply:
yes, scripts largely don't use stun or use the jank immunities, this is the equivalent to the lua script here. I will revert this and will do a script pass in another PR